### ISSUE TITLE
"cockatrice: undefined symbol" after install

### ISSUE THREAD

**Issue by kipahlord13** [Cockatrice/Cockatrice#2696]
<b>OS:</b> Ubuntu 16.04

<b>Cockatrice version:</b> 2.3.17? Unsure, installed May 7, 2017

__________

I just manually installed Cockatrice on Ubuntu 16.04. I updated my distro just earlier today, and I haven't messed with anything that could have broken it, and as  best I could tell from the terminal output, nothing in the install went wrong. However, when I run `cockatrice` to launch it, I get this error message:
`cockatrice: symbol lookup error: cockatrice: undefined symbol: _ZN6google8protobuf8internal13empty_string_E`
and then it exits. What went wrong, and how do I make it work?

**Comment by tooomm** [comment]
You can check your version if you click `Help --> About Cockatrice`

No idea about the symbol...

**Comment by ctrlaltca** [comment]
Just looking at the error i'd say that cockatrice has been compiler with a version of the protobuf library and is trying to run using another (or the same version, but compiled with a different toolchain).
To be more precise it would be nice to have the clean compilation logs and the result of `ldd /path/to/cockatrice`

**Comment by Daenyth** [comment]
Try recompiling cockatrice

**Comment by kipahlord13** [comment]
Update: I seem to have solved the problem, though I'll admit that I'm not quite sure what was happening the whole time. Previously, I had Cockatrice running on 15.04, and I ran it by running `cockatrice` in the command line. When I just installed on 16.04, I figured it would run the same way. But instead, if I just double click the Cockatrice application file in the folder, it runs just fine.

Not quite problem solved, but problem definitely avoided.
